src/extensions/FileDropExtension/FileDropExtensionLoader.ts

```diff
@@ -0,0 +1,11 @@
+import { FileDropExtension } from '@src/extensions/FileDropExtension/FileDropExtension';
+import { Instance } from '@src/Instance';
+import { Config, IModuleLoader, ModuleLoaderTypes } from '@src/types';
+
+export const FileDropExtensionLoader = {
+  type: ModuleLoaderTypes.EXTENSION,
+
+  create: async (instance: Instance) => new FileDropExtension(instance),
+
+  isSupported: ({ config }: { config: Config }): boolean => !!config.fileDrop,
```

Comment:
Also, I think there should be a way of accessing the instance here to do something like:

```typescript
isSupported: ({ config }: { config: Config }): boolean => !!config.fileDrop && instance.player instanceof HTML5Player
```